docs/source/daml-repl/index.rst

```diff
@@ -0,0 +1,121 @@
+.. Copyright (c) 2020 The DAML Authors. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+DAML Repl
+###########
+
+**WARNING:** DAML Repl is an experimental feature that is actively
+being designed and is *subject to breaking changes*.
+We welcome feedback about DAML script on
+`our issue tracker <https://github.com/digital-asset/daml/issues/new>`_
+or `on Slack <https://hub.daml.com/slack/>`_.
+
+DAML Repl allows you to use the :doc:`/daml-script/index` API
+interactively. This is useful for debugging and for interactively
+inspecting and manipulating a ledger.
+
+Usage
+=====
+
+First create a new project based on the ``script-example``
+template. Take a look at the documentation for
+:doc:`/daml-script/index` for details on this template.
+
+.. code-block:: sh
+
+   daml new script-example script-example # create a project called script-example based on the template
+   cd script-example # switch to the new project
+
+Now, build the project and start :doc:`/tools/sandbox`, the in-memory
+ledger included in the DAML SDK. Note that we are starting Sandbox in
+wallclock mode. Static time is not supported in ``daml repl``.
+
+.. code-block:: sh
+
+   daml build
+   daml sandbox --wall-clock-time --port=6865 .daml/dist/script-example-0.0.1.dar
+
+Now that the ledger has been started, you can launch the repl in a
+separate terminal using the following command.
+
+.. code-block:: sh
+
+   daml repl --ledger-host=localhost --ledger-port=6865 .daml/dist/repl-playground-0.0.1.dar
```

Comment:
```suggestion
   daml repl --ledger-host=localhost --ledger-port=6865 .daml/dist/script-example-0.0.1.dar
```
Shouldn't this be the same DAR as above?

Reply:
Good catch!